src/TrackGroup.js

```diff
@@ -143,6 +146,15 @@ OncoTrackGroup.prototype.init = function (container) {
         .attr('text-anchor', 'end')
         .attr('class', _self.prefix + 'track-group-label')
         .text(_self.name);
+    
+    _self.legend = _self.container.append('text')
+        .attr('x', _self.width)
+        .attr('y', -7)
+        .attr('dy', '.32em')
+        .attr('text-anchor', 'end')
+        .attr('class', _self.prefix + 'legend-group-label')
+        .style('fill', 'blue')
```

Comment:
Sure. So I think that styling would have to be done from Portal side though, right?